kamailio/traffic-filter-role.cfg

```diff
@@ -20,6 +20,19 @@ route[FILTER_REQUEST]
 
 route[FILTER_REQUEST_DOMAIN]
 {
+    #!ifdef DISPATCHER_ROLE
+    if (
```

Comment:
@sergey-safarov Howdy, this should not be required...

When the request is received it goes to CLASSIFY_SOURCE which uses logic (effectively the same) to set the flag FLAG_TRUSTED_SOURCE.  This flag is used to skip the filter on the three calls to this route:

FILTER_REQUEST returns if the flag FLAG_TRUSTED_SOURCE is set.
https://github.com/2600hz/kazoo-configs-kamailio/blob/master/kamailio/traffic-filter-role.cfg#L4

The call to FILTER_REQUEST_DOMAIN from HANDLE_OPTIONS and HANDLE_NOTIFY both also check the FLAG_TRUSTED_SOURCE (perhaps we could check it and return at the top of FILTER_REQUEST_DOMAIN for clarity).
https://github.com/2600hz/kazoo-configs-kamailio/blob/master/kamailio/default.cfg#L393
https://github.com/2600hz/kazoo-configs-kamailio/blob/master/kamailio/default.cfg#L426

As such it would appear that this flag is not being set in your environment to cause this change to be required?